tests/subsys/storage/stream/stream_flash/testcase.yaml

```diff
@@ -2,6 +2,10 @@ tests:
   storage.stream_flash:
     platform_whitelist: native_posix native_posix_64
     tags: stream_flash
+storage.stream_flash.dword_wbs:
+    extra_args: DTC_OVERLAY=./unaligned_flush.overlay
```

Comment:
Does this work?  `DTC_OVERLAY` doesn't appear to be defined.  `DTC_OVERLAY_FILE` is.

In any case using a relative path seems unusual (contrast with `OVERLAY_CONFIG=` below).

Reply:
fixed